joark/src/main/kotlin/no/nav/helsearbeidsgiver/pdf/PdfBuilder.kt

```diff
@@ -38,28 +40,36 @@ class PdfBuilder(
         return add(Text(1, LINETEXT, false, false, x, y))
     }
 
-    fun addFootnote(title: String, x: Int = 0, y: Int = 0): PdfBuilder {
-        return add(Text(16, title, false, false, x, y))
+    fun addItalics(title: String, x: Int = 0, y: Int = 0): PdfBuilder {
+        return add(Text(16, title, false, true, x, y))
```

Comment:
Her blir en false om til true. Bare for å dobbelsjekke: Stemmer det?

Reply:
Det stemmer.